### DISCUSSION TITLE
Docker install got 400: Bad Request error

### DISCUSSION THREAD

**Discussion by fishfish77** [kimai/kimai#3398] [Questions & Answers] [answered]
Hello,

I set kimai up in 2 docker containers and if I want to connect to the site throught the browser, I only get an error 400...
Can anyone help me please?

Kind regards!
![Screenshot 2022-07-04 144057](https://user-images.githubusercontent.com/108674352/177157032-9836daea-3473-46cc-8c37-5d22874ec545.png)
![Screenshot 2022-07-04 144143](https://user-images.githubusercontent.com/108674352/177157181-7a104951-869c-4e48-9072-35fca5172cf1.png)

**Comment by kevinpapst** [comment, marked as answer]
This is an environment issue and not a Kimai problem.
If you need support with hosting issues please ask in the forum of your choice, e.g. https://serverfault.com

For docker related issues please go to https://github.com/tobybatch/kimai2

**Comment by mm86133** [comment]
Same issue here.

Not sure how that is an env issue if that is copy/paste docker-compose.

Ubuntu 18, all other dockers working fine.

**Comment by cvural** [comment]
Make sure TRUSTED_HOSTS has the IP address or your reverse proxy in the list. Like this ...

- TRUSTED_HOSTS=nginx,localhost,**192.168.1.10,time.example.net**,127.0.0.1

**Reply by ThijmenGThN** [reply]
✨Solved it for me, thank you for the comment.
